rtic-monotonics/src/stm32.rs

```diff
@@ -294,21 +294,13 @@ macro_rules! make_timer {
             }
 
             fn clear_compare_flag() {
-                $timer.sr().modify(|r| r.set_ccif(1, false));
+                // Do nothing, because the flags are cleared below in on_interrupt
```

Comment:
This is incorrect, this flag is for the user triggered interrupt, the other one is for the half-overflow interrupt. Please do not remove this line.